profiles/Chuno-stealth.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
All of these YAML updates (minus the whirlwind specific changes) would be a good separate PR